src/common/components/featureSupportLevels/utils.ts

```diff
@@ -47,11 +47,7 @@ export const getLimitedFeatureSupportLevels = (
     }
     return ret;
   } catch (err) {
-    captureException(
-      err,
-      `Failed to get cluster ${cluster.id} feature support levels`,
-      Sentry.Severity.Warning,
-    );
+    errorHandler(err, `Failed to get cluster ${cluster.id} feature support levels`);
```

Comment:
IMHO logging things to the console and passing no `errorHandler` is pretty much the same. Both options are noops, maybe the latter would be useful in local development, but the first could impact the app's performance.
If console.logging is the actual consumer's intention, I'd let them do that explicitly rather than doing that for them implicitly.

Reply:
no need to write IMHO in reviews of my code ;)